symbolic_pymc/tensorflow/meta.py

```diff
@@ -340,10 +340,13 @@ def process_node_def(cls, node_def, op_def, obj=None):
     @classmethod
     def _metatize(cls, obj):
         """Fix inputs from TF"""
-        obj.input = obj._reconstruct_sequence_inputs(obj.op_def,
-                                                      obj.inputs,
-                                                      obj.node_def.attr)
-    
+        new_input = obj._reconstruct_sequence_inputs(obj.op_def,
+                                                     obj.inputs,
+                                                     obj.node_def.attr)
+        new_args = [getattr(obj, s) for s in getattr(cls, "__slots__", [])]
+        new_args[-2] = new_input
```

Comment:
This shouldn't have to rely on the order of elements in `__slots__`, nor use a value that is immediately replaced.  Instead, you could check for when `s == "inputs"` in the list comprehension and use `new_input` in the first place.

Reply:
Fixed in latest commit.